aten/src/ATen/native/TensorCompare.cpp

```diff
@@ -4,8 +4,28 @@
 #include "ATen/ExpandUtils.h"
 #include "ATen/NativeFunctions.h"
 
-namespace at {
-namespace native {
+namespace {
+template <typename scalar_t>
+void where_cuda(
+    at::Tensor& ret,
+    const at::Tensor& condition,
+    const at::Tensor& self,
+    const at::Tensor& other) {
+  at::CPU_tensor_apply4<scalar_t, uint8_t, scalar_t, scalar_t>(
```

Comment:
why is `CPU_tensor_apply` inside `where_cuda`?

Reply:
@ezyang is too quick on the trigger! @goldsborough can you fix this? I think the name of the function is just wrong (should be `where_cpu` instead of `where_cuda`)

Reply:
My bad, will follow up